test_conformance/compiler/main.cpp

```diff
@@ -89,6 +89,7 @@ test_definition test_list[] = {
 
     ADD_TEST_VERSION(pragma_unroll, Version(2, 0)),
 
+    ADD_TEST_VERSION(features_macro, Version(2, 0)),
```

Comment:
This should be 3.0, right?

```suggestion
    ADD_TEST_VERSION(features_macro, Version(3, 0)),
```